### ISSUE TITLE
Blocking Issue: Missing organization(s) for Github user

### ISSUE THREAD

**Issue by pcullin-ratio** [Shippable/support#1139]
One of the users on my team was able to create his Shippable account using Github credentials, but only one of the Github organizations to which he is a member shows up in the Shippable dashboard. I'm a member of the same organization he is that is not showing up in his account, but it does display for me. We are both "owners" of the organization in Github. In his Shippable dashboard, both public and private repos are enabled.

Is there a way to force Shippable to pick up the organization? Neither one of us can find any way to configure this and it is super frustrating. I really need him to be able to access the repos in this organization, and his lack of access is blocking our progress at this point. Please help!

**Comment by manishas** [comment]
@pcullin-ratio can you give us the github username for this user and also the org that doesn't show up on his dashboard? Thanks!

**Comment by pcullin-ratio** [comment]
His account is github/bbronske and the organization is called LabyrinthLLC.

I just resynched my account, and now the LabyrinthLLC organization is gone for me too. This is our main build target organization, so obviously not a good thing ☺


Paul Cullin //  Lead Software Engineer
RATIO | 71 Columbia St. Suite 200, Seattle WA 98104
WeAreRatio.com<http://weareratio.com/> | o 206.281.8240 | c 425.922.2395

**Comment by manishas** [comment]
@pcullin-ratio I see you as a collaborator in 2 orgs, but not the one you mention. Are you sure someone didn't remove permissions on GitHub by mistake?

**Comment by pcullin-ratio** [comment]
Yes, I’m sure – looking at the organization right now in Github. Both bbronske and pcullin-ratio are in the Owners group, which provides full access to all repositories in the organization, along with administrative access to the organization itself.

Is it possible Shippable is having issues communicating with Github fully?

Thanks for your quick response ☺

**Comment by manishas** [comment]
we'll take a look. i don't think we've had issues with github, but will investigate.

**Comment by pcullin-ratio** [comment]
Just found the issue, but not sure how this happened. In Github, I went to the Organization Applications settings for the LabyrinthLLC organization, and saw that prod-shippable was indeed there in the list of Authorized Applications. When I clicked on the “View” button for prod-shippable, it showed a list of “Organization access” and somehow LabyrinthLLC was not authorized. So all I had to do was click the Authorize button, and now both myself and bbronske can see the LabyrinthLLC organization in our Shippable dashboards after re-synching.

Problem is, I have no idea how the organization became unauthorized. Although there are a couple other Github users with access to modify organization settings, bbronske and myself are the only ones who would even know how to access this in Github, and neither him nor I have changed these settings manually.

Very odd…

Thank you for the assistance. I think we’re back in business for now.
